domain/iam.go

```diff
@@ -21,4 +21,5 @@ type IAMManager interface {
 // IAMClient interface
 type IAMClient interface {
 	GetUser(id string) (interface{}, error)
+	IsActiveUser(id string) (bool, error)
```

Comment:
@singhvikash11 If I understand this correctly, this function will be used to check if the user is active or not on demand? Does `GetUser` return this status? Any reason for us to have this as a separate method?

Reply:
Having a separate `IsActiveUser ` function will allow users to write some complex custom functions to connect external systems etc. If you say we can move into `GetUser`